docs/Validation-and-Serialization.md

```diff
@@ -658,7 +658,53 @@ fastify.setErrorHandler(function (error, request, reply) {
 })
 ```
 
-If you want custom error response in schema without headaches and quickly, you can take a look at [here](https://github.com/epoberezkin/ajv-errors)
+If you want custom error response in schema without headaches and quickly, you can take a look at [here](https://github.com/epoberezkin/ajv-errors).
+
+Below is an example to add custom error messages for each property. Inline comments describe how you can configure your schema to show different error message for each case
+
+```js
+const Ajv = require('ajv')
+const AjvErrors = require('ajv-errors')
```

Comment:
This is non-idiomatic. The module exports a regular function, not a "constructor function".
```suggestion
const ajvErrors = require('ajv-errors')
```